src/nft/components/collection/CollectionAsset.tsx

```diff
@@ -145,13 +146,28 @@ export const CollectionAsset = ({
             rarityLogo={rarityLogo}
           />
         )}
-        {assetMediaType === AssetMediaType.Image ? (
-          <Card.Image />
-        ) : assetMediaType === AssetMediaType.Video ? (
-          <Card.Video shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        ) : (
-          <Card.Audio shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
-        )}
+        <MouseoverTooltip
```

Comment:
I don't think this is the correct way to implement this. I think it should be implemented in a similar fashion as the added and removed to bag popover. If you look this has some weird implications  - it causes the rarity tooltips to become slightly misaligned
![Screen Shot 2022-12-01 at 3 35 22 PM](https://user-images.githubusercontent.com/9094792/205154157-320aa939-52f8-41e5-bf23-0401091494ee.png)

Reply:
hmm i don't see whats misaligned about it? 

it can't be implemented the same way as added /removed because it needs to wrap the component (image) where the mouseover triggers the tooltip.